src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -255,13 +255,10 @@ function compute_diffusive_fluxes(
     )
     Ic = CCO.InterpolateF2C()
 
-    @. aux_tc_f.ρ_ae_KH = IfKH(aeKH) * ρ_f
-    @. aux_tc_f.ρ_ae_KM = IfKM(aeKM) * ρ_f
-
-    aeKHq_tot_bc = -surf.ρq_tot_flux / a_en[kc_surf] / aux_tc_f.ρ_ae_KH[kf_surf]
-    aeKHh_tot_bc = -surf.ρe_tot_flux / a_en[kc_surf] / aux_tc_f.ρ_ae_KH[kf_surf]
-    aeKMu_bc = -surf.ρu_flux / a_en[kc_surf] / aux_tc_f.ρ_ae_KM[kf_surf]
-    aeKMv_bc = -surf.ρv_flux / a_en[kc_surf] / aux_tc_f.ρ_ae_KM[kf_surf]
+    aeKHq_tot_bc = FT(0)
+    aeKHh_tot_bc = FT(0)
+    aeKMu_bc = FT(0)
+    aeKMv_bc = FT(0)
```

Comment:
How is this possible?

Reply:
these are only here for diagnostics as we apply a lower bc on the SGS flux given by the flux from SF.jl. what ever is set here is overwritten later